test/examples_public_private_test.go

```diff
@@ -11,16 +11,22 @@ func TestExamplesPublicPrivateCWLogs(t *testing.T) {
 
 	terraformOptions := &terraform.Options{
 		TerraformDir: "../examples/public_private_flow_logs",
+		Vars: map[string]interface{}{
+			"vpc_flow_logs": map[string]interface{}{
+				"name_override":        "test",
+				"log_destination_type": "cloud-watch-logs",
+				"kms_key_id":           nil,
+			},
+		},
 	}
 
 	defer terraform.Destroy(t, terraformOptions)
 	terraform.InitAndApply(t, terraformOptions)
 	terraform.ApplyAndIdempotent(t, terraformOptions)
 
-	publicTagsLength := terraform.Output(t, terraformOptions, "public_subnets_tags_length")
-	assert.Equal(t, "2", publicTagsLength)
-	privateTagsLength := terraform.Output(t, terraformOptions, "private_subnets_tags_length")
```

Comment:
Was this removed intentionally?

Reply:
good catch. meant to add those back after running the test i intended. adding now, 1s